app/src/main/java/com/framgia/fsalon/utils/Utils.java

```diff
@@ -45,4 +55,27 @@ public static void disableShiftMode(BottomNavigationView view) {
             Log.e("BNVHelper", "Unable to change value of shift mode", e);
         }
     }
+
+    public static int createTimeStamp(@SchedulerViewModel.TabFilter int tab) {
+        Calendar calendar = Calendar.getInstance();
+        switch (tab) {
+            case TAB_TODAY:
+                return (int) (calendar.getTime().getTime() / 1000);
+            case TAB_TOMORROW:
+                calendar.add(Calendar.DAY_OF_YEAR, 1);
+                return (int) (calendar.getTime().getTime() / 1000);
+            case TAB_YESTERDAY:
+                calendar.add(Calendar.DAY_OF_YEAR, -1);
+                return (int) (calendar.getTime().getTime() / 1000);
+            default:
+                break;
+        }
+        return OUT_OF_INDEX;
+    }
+
+    public static String convertTime(Date date) {
+        if (date == null) return null;
+        SimpleDateFormat sfd = new SimpleDateFormat("hh:mm:ss");
```

Comment:
WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.